playbooks/generic-deploy_airship.yml

```diff
@@ -36,6 +36,13 @@
       tags:
         - install
 
+- hosts: airship-openstack-compute-workers
+  gather_facts: yes
+  any_errors_fatal: true
+  roles:
+    - role: airship-configure-worker
+      when: redeploy_osh_only is defined and (redeploy_osh_only | bool)
```

Comment:
is redeploy_osh_only needed? I thought we have a different tag for running this purpose and we are deprecating this env variable?

Reply:
> is redeploy_osh_only needed? I thought we have a different tag for running this purpose and we are deprecating this env variable?

if redeploy_osh is undefined then  will go to all the tasks to check in deploy osh ? and I am running this only compute-workers where as other check on  caasp workers.